src/main/resources/rulesets/checkstyle/rules.xml

```diff
@@ -52,6 +52,15 @@
   <module name="org.openhab.tools.analysis.checkstyle.RequireBundleCheck">
       <property name="severity" value="error" />
    </module>
+   
+   <module name="org.openhab.tools.analysis.checkstyle.AboutHtmlCheck">
+     <property name="severity" value="error" />
+     <!-- You can give a path to a valid about.html file with the property "validAboutHtmlFilePath", e.g.
+          <property name="validAboutHtmlFilePath" value="..." />
+          The processed about.html files will be compared with this file.
+          If you don't provide such property or the path is incorrect,
```

Comment:
Yes, if someone wants to perform the AboutHtmlCheck, he has to provide a path to a valid about.html file. Otherwise, this check will not be executed.